integrations/cypress/cypress/support/utils/config.ts

```diff
@@ -10,35 +10,36 @@ interface Config {
   clientID: string;
   domain: string;
   sdk: Auth0SDKs
+  auth0RunningPort: number;
+  auth0SimulatorPort: number;
 }
 
 export function getConfig(): Config {
-  let sessionCookieName = Cypress.env('auth0SessionCookieName') ?? 'appSession';
-  let cookieSecret = Cypress.env('auth0CookieSecret');
-  let audience = Cypress.env('audience');
-  let connection = Cypress.env('connection') ?? 'Username-Password-Authentication';
-  let scope = Cypress.env('scope') ?? 'openid profile email offline_access';
-  let clientSecret = Cypress.env('auth0ClientSecret');
-  let clientID = Cypress.env('clientID') ?? 'YOUR_AUTH0_CLIENT_ID';
-  let domain = Cypress.env('domain') ?? 'localhost:4400';
-  let sdk = (Cypress.env('AUTH0_SDK') ?? 'auth0_react') as Auth0SDKs;
-
+  let auth0RunningPort = Cypress.env('AUTH0_RUNNING_PORT') ?? 4400;
```

Comment:
```suggestion
  let auth0RunningPort = Cypress.env('AUTH0_SIMULATOR_PORT') ?? 4400;
```

I think we only need 1 of these variables and callit `AUTH0_SIMULATOR_PORT`

Reply:
Sure, i think there won't be a reason a user would need to change this port. I will update it :+1:

Reply:
I have fixed this :+1: